code/modules/client/preferences.dm

```diff
@@ -675,6 +675,8 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 					dat += " |"
 				if(category == gear_tab)
 					dat += " <span class='linkOff'>[category]</span> "
+				if(category == "Donator" && (!LAZYLEN(GLOB.patrons) || !CONFIG_GET(flag/donator_items)))
+					continue
```

Comment:
This exemption has to happen earlier. This position disrupts existing control flow.